app/src/main/java/com/example/foody/recipe_search/domain/RecipesSearchRepository.kt

```diff
@@ -11,4 +13,5 @@ import com.example.foody.shared.domain.model.RecipeInfo
 // In this interface we define a search function which returns a list of recipes
 interface RecipesSearchRepository {
     suspend fun search(searchTerm: String) : List<RecipeInfo>
+    suspend fun singleRandomMeal() : List<RecipeInfo>
```

Comment:
We should remove all unrelated to this PR scope

Reply:
? I thought I removed this.  I've missed it apparently. Oki

Reply:
I did! I did taw a puttycat! ;)
That shouldn't be there